packages/server/src/core/degree_status/postprocessing.rs

```diff
@@ -14,15 +11,7 @@ impl DegreeStatus {
         let completed_english_content_courses_count = self
             .course_statuses
             .iter()
-            .filter(|course_status| {
-                course_status
-                    .course
-                    .tags
-                    .clone()
-                    .unwrap_or_default()
-                    .contains(&Tag::English)
-                    && course_status.completed()
```

Comment:
`&& course_status.completed()` was also removed, is this a mistake?

Reply:
yes :')